playbooks/generic-build_images.yml

```diff
@@ -11,10 +15,29 @@
         - always
         - imagebuilder
         - upstream_patching
-  roles:
-    - role: suse-build-images
-      tags:
-        - imagebuilder
+
+    # The certificate generated before also needs to be known by the osh-deployer
+    - name: If developer mode, configure osh-deployer to include certs
+      include_role:
+        name: setup-caasp-workers
+        tasks_from: copy-certificates
+      when:
+        - developer_mode
+
+    - name: If developer mode, install docker
+      include_role:
+        name: install-docker
+        apply:
+          tags:
+            - docker
```

Comment:
Where's the check for `developer_mode` ?

Reply:
It's an env variable DEVELOPER_MODE. That env var is loaded from the common vars. 
It hasn't changed for a while, and, for once... is in the readme :)

Reply:
FYI this gets into the playbooks from here: https://github.com/SUSE-Cloud/socok8s/blob/master/vars/common-vars.yml#L3

Reply:
I understand that. I was missing `when:  - developer_mode` condition, I don't know how I missed that